webdriver-spec.html

```diff
@@ -432,6 +432,13 @@
      <li><dfn><a href="https://www.w3.org/TR/page-visibility/#dom-document-hidden">Document <code>hidden</code> attribute</a></dfn>
    </ul>
 
+ <dt>Selenium
+ <dd>The following functions are defined within
+  the <a href="http://seleniumhq.org">Selenium</a> project.
+  <ul>
+   <!-- bot.dom.getVisibleText --> <li><dfn><a href="https://github.com/SeleniumHQ/selenium/blob/1721e627e3b5ab90a06e82df1b088a33a8d11c20/javascript/atoms/dom.js#L979"><code>bot.dom.getVisibleText</code></a></dfn>
+   <!-- bot.dom.isShown --> <li><dfn><a href="https://github.com/SeleniumHQ/selenium/blob/1721e627e3b5ab90a06e82df1b088a33a8d11c20/javascript/atoms/dom.js#L548"><code>bot.dom.isShown</code></a></dfn>
+  </ul>
```

Comment:
Possibly include normative reference to SHAs in the text too? They’re only in the links now.